addons/sourcemod/scripting/MyJailbreak/weapons.sp

```diff
@@ -291,7 +291,13 @@ public void OnConfigsExecuted()
 	g_aPrimary = CreateArray(128);
 	g_aSecondary = CreateArray(128);
```

Comment:
I think we need to close these handles as well.

Reply:
here https://github.com/shanapu/MyJailbreak/pull/333